benchmark/inference/utils.py

```diff
@@ -1,8 +1,8 @@
 import os.path as osp
 
 import torch
-from hetero_gat import HeteroGAT
-from hetero_sage import HeteroGraphSAGE
+from inference.hetero_gat import HeteroGAT
```

Comment:
Shall we move to `benchmark/utils` or something? A bit weird to have models be defined in `inference` only.